### PR TITLE
jenkins: Backup jenkins configs

### LINKED ISSUE #507: backport general jenkins configs

### PR DESCRIPTION
**Pull request by GabyCT**
This PR backup some general jenkins configs.

Fixes #507

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

**Review by chavafg — DISMISSED** [review]

File: jenkins/jobs/kata-containers-2.0-tests-metrics-ubuntu-20-04-PR/config.xml [hunk 1]
```diff
@@ -110,8 +110,16 @@ set -ex
...
+#sudo service docker restart
+#sudo systemctl status docker
+#whereis docker
+
 curl -OL https://raw.githubusercontent.com/kata-containers/tests/main/.ci/ci_entry_point.sh
 export CI_JOB=&quot;METRICS&quot;
+#sudo ctr task kill $(sudo ctr task ls -q)
+#sudo ctr tasks rm -f $(sudo ctr task list -q)
+#sudo ctr tasks rm -f $(sudo ctr task list -q)
+#sudo ctr c rm $(sudo ctr c list -q)
```
Comment (dborquez) [inline jenkins/jobs/kata-containers-2.0-tests-metrics-ubuntu-20-04-PR/config.xml:122]: Could you remove the commented lines?

File: jenkins/jobs/kata-containers-2.0-metrics-ubuntu-20-04-PR/config.xml [hunk 2]
```diff
@@ -108,7 +108,16 @@ set -ex
...
 sudo umount /usr/share/kata-containers/kata-containers.img || true
 sudo rm -f /usr/share/kata-containers/kata-containers.img
+#sudo rm -rf /var/lib/containerd/
+#sudo rm -rf /var/lib/docker/
+#sudo apt-get remove -y docker docker-engine docker.io containerd runc
+#sudo apt-get update
+#sudo apt-get install -y docker-ce docker-ce-cli containerd.io docker-compose-plugin
 
+#whereis docker
+#whereis containerd
```
Comment (dborquez) [inline jenkins/jobs/kata-containers-2.0-metrics-ubuntu-20-04-PR/config.xml:118]: Could you remove the commented lines?

**Review by dborquez — CHANGES_REQUESTED** [review]
Thank you @GabyCT , just a couple of changes.

File: jenkins/jobs/kata-containers-2.0-metrics-ubuntu-20-04-PR/config.xml [hunk 3]
```diff
@@ -170,12 +179,12 @@ sudo chgrp -R $(id -ng) ${WORKSPACE}</script>
     </org.jenkinsci.plugins.credentialsbinding.impl.SecretBuildWrapper>
     <hudson.plugins.build__timeout.BuildTimeoutWrapper plugin="build-timeout@1.20">
       <strategy class="hudson.plugins.build_timeout.impl.NoActivityTimeOutStrategy">
-        <timeoutSecondsString>900</timeoutSecondsString>
```
Comment (dborquez) [inline jenkins/jobs/kata-containers-2.0-metrics-ubuntu-20-04-PR/config.xml:173]: I assume the timeoutSecondsString is updated by observation?

Comment (GabyCT) [reply jenkins/jobs/kata-containers-2.0-metrics-ubuntu-20-04-PR/config.xml:173]: @dborquez yes it is

**Review by dborquez — APPROVED** [review]

**Pull request merged**